client/web/src/enterprise/batches/create/NewCreateBatchChangeContent.tsx

```diff
@@ -0,0 +1,208 @@
+                    <ul>
+                        <li>You prefer to execute the spec on your personal system</li>
+                        <li>
+                            You enjoy reminiscing about the "good old days" of Visual Basic, dial-up internet, and
+                            burning CDs, you still use Yahoo as your search engine of choice, or you're still not sold
+                            on this "newfangled MySpace thing" all the kids are using these days
+                        </li>
+                        <li>
+                            You strongly relate to{' '}
+                            <a href="https://i.imgur.com/91sn32Q.jpeg" rel="noopener noreferrer" target="_blank">
+                                this image
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+                <div className="w-50 pl-3">
+                    <h3 className="d-flex align-items-center">
+                        On Sourcegraph <span className="badge badge-info text-uppercase ml-1">Experimental</span>
+                    </h3>
+                    <p>
+                        Use{' '}
+                        <a
+                            href="https://docs.sourcegraph.com/@batch-executor-mvp-docs/batch_changes/explanations/executors"
+                            rel="noopener noreferrer"
+                            target="_blank"
+                        >
+                            Sourcegraph executors
+                        </a>{' '}
+                        to run the spec and preview your batch change.
+                    </p>
+                    <NamespaceSelector
+                        settingsCascade={settingsCascade}
+                        selectedNamespace={selectedNamespace}
+                        onSelect={setSelectedNamespace}
+                    />
+                    <button
+                        type="button"
+                        className="btn btn-primary mb-3"
+                        onClick={submitBatchSpec}
+                        disabled={isLoading === true}
+                    >
+                        Run batch spec
+                    </button>
+                    {isErrorLike(isLoading) && <ErrorAlert error={isLoading} />}
+                    <hr className="mb-3" />
+                    <p>
+                        This is the <strong>recommended</strong> way to execute a batch spec. Choose this option if:
+                    </p>
+                    <ul>
+                        <li>You're trying out batch changes for the first time</li>
+                        <li>
+                            You don't have <span className="text-monospace">src-cli</span> set up
+                        </li>
+                        <li>Your batch spec takes a lot of time or resources to execute</li>
+                        <li>You swoon anytime you hear the phrase "cloud infrastructure" ☁️☁️☁️</li>
```

Comment:
```suggestion
                        <li>You swoon anytime you hear the phrase "cloud infrastructure" >>☁️☁️☁️<<</li>
```

What is this mysterious character? 😆

Reply:
Hahaha idk why they look like this in monospace fonts but it's the cloud emoji!! ☁️ ☁️ ☁️ 

![image](https://user-images.githubusercontent.com/8942601/128543711-f2b2306c-2108-460b-8ae7-86bb264ec867.png)